### ISSUE TITLE
[page title] Unable to distinguish content on page

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#157]
The page title is always "Open Referral UK" and should describe the page content.

URL: https://openreferraluk.org/about-standard

Pattern to use
[H1] - Open Referral UK

**Comment by adamgparsons** [comment]
This appears to be fixed on the site as part of the work in #209. When you go to the [about the standard](https://openreferraluk.org/about-standard) page the page title is "Contact us | Open Referral UK". This is similar on other pages.

**Comment by kingdomlevel** [comment]
Should, in theory, be closed once 209 merges due to [the new PageTitle component](https://github.com/WeAreSnook/OR-UK-frontend/pull/209#discussion_r672334635) (nice work!).

**Comment by UsmanAfzal** [comment]
Checked and approved
